cmd/postgres-operator/main.go

```diff
@@ -134,7 +134,12 @@ func main() {
 
 	features := feature.NewGate()
 	assertNoError(features.Set(os.Getenv("PGO_FEATURE_GATES")))
-	log.Info("feature gates enabled", "PGO_FEATURE_GATES", features.String())
+
+	ctx = feature.NewContext(ctx, features)
+	// This logs just the feature gates as set by the user
+	log.Info("feature gates set during deployment", "PGO_FEATURE_GATES", feature.ShowAssigned(ctx))
+	// This logs the feature gates that are enabled, including gates that are on by default
+	log.Info(fmt.Sprintf("feature gates enabled: %s", feature.ShowEnabled(ctx)))
```

Comment:
🤔 Usually, a `fmt` indicates we should have key/value pairing. Maybe a single log message? 🤷🏻 

```suggestion
	log.Info("feature gates",
		// These are set by the user
		"PGO_FEATURE_GATES", feature.ShowAssigned(ctx),
		// These are enabled, including features that are on by default
		"enabled", feature.ShowEnabled(ctx))
```

🤔 🤔 are the letters `PGO_FEATURE_GATES` useful at this moment? `assigned` and `enabled` attributes? 🤔

Reply:
Single log message makes sense to me; I like `PGO_FEATURE_GATES` since that's the env var that the user sets, and I would almost drop it (since the user could just check the deployment) but I like the convenience of getting all the info at one point.

I do want to update the documentation we have [here](https://access.crunchydata.com/documentation/postgres-operator/latest/installation/feature-gates) to add "look at your logs to see what's enabled."